guide/payments/receive.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: An introduction to how receiving bitcoin works.
+nav_order: 3
+parent: Payments
+permalink: /guide/payments/receive/
+main_classes: -no-top-padding
+# image: /assets/images/guide/payments/receive/header.svg
+---
+
+# Receiving bitcoin
+
+<!--
+
+Editor's notes
+
+This page should cover what to do when receiving bitcoin, how to share and copy addresses etc.
+
+- What addresses to support (native segwit by default)
+- How to present QR codes (allow for specifying an amount)
+- Generating a new address 
+
+-->
+
+For someone to receive bitcoin, they must first provide the sender with a payment destination. They can do this by generating an address within their Bitcoin wallet application. A new address should be generated for every payment, as addresses should only be used once. This helps safeguards the receiver’s privacy.
+
+<!-- 
+Update glossary
+
+Each address that a Bitcoin application generates has an accompaniying private key. This private key allows the receiver to unlock the bitcoin that is sent to the address when the receiver chooses to spend it.
+
+Although addresses are not a problem to share publically, their accompanying private keys which typically generated from a single recovery phrase must be kept secure and private.
+-->
+
+## Preparing the payment request
```

Comment:
```suggestion
## Requesting bitcoin
```

Reply:
i think having the full term payment* request in there is more useful than not.